android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java

```diff
@@ -74,11 +76,12 @@
 
     private String searchText = "";
     private SearchView searchView;
+    final ArrayList<String> dateList = new ArrayList<>();
 
-    private DayScheduleAdapter bookMarksListAdapter;
+    private GlobalSearchAdapter bookMarksListAdapter;
```

Comment:
@shreyasnbhat Adding bookmarks in Global Search results is not a good idea. Bookmarks are specific to particular user and global search should give results from all over the app.